src/qibo/transpiler/unroller.py

```diff
@@ -85,16 +85,7 @@ def from_gate(cls, gate):
 
 # TODO: Make setting single-qubit native gates more flexible
 class Unroller:
-    """Translates a circuit to native gates.
-
-    Args:
-        native_gates (:class:`qibo.transpiler.unroller.NativeGates`): native gates to use
-            in the transpiled circuit.
-        backend (:class:`qibo.backends.Backend`): backend to use for gate matrix.
-
-    Returns:
-        (:class:`qibo.models.circuit.Circuit`): equivalent circuit with native gates.
-    """
+    """Decomposes a circuit to native gates."""
```

Comment:
Where did the docstring go?

Reply:
It has been moved to the docstring of `__init__` and `__call__`.

Reply:
[8e903f0](https://github.com/qiboteam/qibo/pull/1531/commits/8e903f02983393a18e42d29a32165f053c33c7d9)